src/middleware/cache.js

```diff
@@ -1,5 +1,10 @@
 module.exports = (req, res, next) => {
-	res.FT_NO_CACHE = 'max-age=0, no-cache, must-revalidate';
+	if (res.locals.flags && res.locals.flags.noStoreCache) {
+		res.FT_NO_CACHE = 'max-age=0, no-cache, no-store, must-revalidate';
+	} else {
+		res.FT_NO_CACHE = 'max-age=0, no-cache, must-revalidate';
```

Comment:
Going with the new way as the default, as we've fixed all the tests etc. in dependent apps.